python/utils.py

```diff
@@ -100,9 +100,7 @@ def py_sampleProfile(intensity, Rmin, dR, nxy, dxy, udat, vdat, dRA=0., dDec=0.,
                  bounds_error=False, assume_sorted=True)
     intensmap = f(x_meshgrid)
 
-    f_center = interp1d(gridrad, intensity, kind='linear', fill_value='extrapolate',
-                 bounds_error=False, assume_sorted=True)
-    intensmap[int(nrow/2), int(ncol/2)] = f_center(0.)
+    intensmap[int(nrow / 2), int(ncol / 2)] = central_pixel(intensity, Rmin, dR, dxy)
```

Comment:
Careful with python 2 vs 3. Do you actually want integer division like `ncol // 2`?

Reply:
yes, I should fix this